TheMovieDb/Search/SearchResultViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchResultViewController.swift
+//  TheMovieDb
+//
+//  Created by Alex on 23/11/21.
+//
+
+import UIKit
+
+final class SearchResultViewController: UIViewController {
+    @IBOutlet private weak var searchResultTableView: UITableView!
+
+    let searchViewModel = SearchViewModel()
```

Comment:
Maybe it would be better if we inject this object to the controller, instead of directly coupling both implementations.

Reply:
Got it.